pkg/ddl/ddl.go

```diff
@@ -1285,6 +1407,106 @@ func (d *ddl) SwitchMDL(enable bool) error {
 	return nil
 }
 
+// SwitchDDLVersion switch ddl version.
+func (d *ddl) SwitchDDLVersion(version int64) error {
```

Comment:
Are there tests that run this function in parallel?
The following case：
session1：
1. get current version：version1
2. exec select SQL
3. exec migration2DDLVersion
4. store version to **version2**

session2：
1. get current version：version1
2. exec select SQL
3. exec migration2DDLVersion
4. store version to **version1**

Reply:
The current version is bool type and has been judged, so it is not affected.